uint256.go

```diff
@@ -144,18 +144,34 @@ func (z *Int) WriteToSlice(dest []byte) {
 	}
 }
 
+// PutUint256 writes all 32 bytes of z to the destination slice, including zero-bytes.
+// If dest is larger than 32 bytes, z will fill the first parts, and leave
+// the end untouched.
+// Note: The dest slice must be at least 32 bytes large, otherwise this
+// method will panic. The method WriteToSlice, which is slower,  should be used
+// if the destination slice is smaller or of unknown size.
+func (z *Int) PutUint256(dest []byte) {
```

Comment:
I think we should add
```
_ = dest[31]
```
so that there is only 1 bound check at the beginning, otherwise, before each PutUint64, there will be a bound check.

Reply:
Sure. I couldn't see any tangible benefit on a bench, but it doesn't hurt